crates/biome_fs/src/fs.rs

```diff
@@ -404,7 +403,7 @@ pub enum ErrorKind {
 impl console::fmt::Display for ErrorKind {
     fn fmt(&self, fmt: &mut console::fmt::Formatter) -> io::Result<()> {
         match self {
-            ErrorKind::CantReadFile(_) => fmt.write_str("Biome couldn't read the file"),
+            ErrorKind::CantReadFile(_) => fmt.write_str("Cannot read file"),
```

Comment:
Since we're updating the diagnostics :)
```suggestion
            ErrorKind::CantReadFile(_) => fmt.write_str("Can't read the file."),
```

We should strive to end the sentences with a full stop.

Reply:
Sounds fair